### ISSUE TITLE
[Bug] safe_pcap_next ERROR: Invalid packet length in send_packets.c:get_next_packet() 

### ISSUE THREAD

**Issue by cgi1** [appneta/tcpreplay#565]
```
# sudo tcpreplay --multiplier=1 -i lo 2019-05-23.pcap
Warning: Unable to process unsupported DLT type: BSD loopback (0x0)              
safe_pcap_next ERROR: Invalid packet length in send_packets.c:get_next_packet() 
method: PF_PACKET send()
line 1068: 75674 is greater than maximum 65549       
```

Than the program simply exit.

The code of the last warning is [exactly here](https://github.com/appneta/tcpreplay/pull/491/commits/68f67b1a3a4d319543692afb5bd5b191ec984287#diff-ebbd90b295f3fb69d024064451ec254eR160) and has been added as a response to [another bug](https://github.com/appneta/tcpreplay/issues/484#issuecomment-431134567).

**To Reproduce**
1. Had a fully working version of tcpreplay v.4.2.6 from tarball where some warnings came up, but the replay in general worked. 
2. Than upgraded to v4.3.2, new warning and now the program simply quits.

**Expected behavior**
Be able to read and replay of packages > 65549 (which does not looks like the correct value, the resolution for the other bug where a max len of 262144)

**System (please complete the following information):**
 - Ubuntu 18.04
 - Tcpreplay Version 4.3.2

**Comment by fklassen** [comment]
Can you supply a packet capture? Easiest is to rename to `*pcap.txt` and attach.

**Comment by cgi1** [comment]
Unfortunately I can not post it public, becaues this is closed data. I can upload it and send you a link via mail, if this is suitable for you?

**Comment by fklassen** [comment]
I believe this issue is resolved int #571 which significantly increases maximum snaplen. Closing for now. Reopoen if reproduced, and will get pcap file from you.
